<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flash Bootloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="button_choose">
       <property name="text">
        <string>Choose file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_size">
       <property name="text">
        <string>File size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_content">
       <property name="text">
        <string>File content:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_can_message">
       <property name="text">
        <string>send CAN-message</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
